src/j1client.js

```diff
@@ -314,6 +314,9 @@ const UPDATE_ALERT_RULE = gql`
       name
       description
       version
```

Comment:
I see a lot of renaming of `version` to `specVersion`. Is this `version` still relevant?

Reply:
Yes. `version` is now the actual revision number of the rule itself. `specVersion` is for managing breaking changes to the rule schema.